pdfPreviewInMsg/background.js

```diff
@@ -0,0 +1,166 @@
+/*
+----------------------------------------------------------------
+This extension for Thunderbird was made by : e-gaulue & rholeczy
+                        © 2021
+----------------------------------------------------------------
+*/
+var initialized;
+
+
+/**
+ * Preview a pdf in the mail.
+ * 
+ * @param {*} pdf 
+ * @param {*} canvas 
+ * @param {*} nbP 
+ * @param {*} tabid 
+ * @param {*} filename 
+ * @param {*} messageid 
```

Comment:
Could you provide short descriptions of each parameter and its type? If you cannot pull my updated sources, modify them and push your changes, than just comment here what I should add there.